data/presets/public_transport/station.json

```diff
@@ -14,8 +14,12 @@
         "{@templates/internet_access}",
         "air_conditioning",
         "baby_feeding",
+        "changing_table",
         "gnis/feature_id-US",
         "level",
+        "toilets",
+        "toilets/wheelchair",
+        "toilets/menstrual_products_poi",
```

Comment:
this fields appearing say at tram stations are surprising, but still, with moreFields it should be fine, I guess?